backend/module/chaos/serverexperimentation/rtds/mock_experiment_store.go

```diff
@@ -38,5 +39,8 @@ func (fs *mockExperimentStore) GetExperimentRunDetails(ctx context.Context, id u
 	return nil, nil
 }
 
+func (fs *mockExperimentStore) RegisterTransformation(typeUrl string, transformation func(config *experimentstore.ExperimentConfig) ([]*experimentation.Property, error)) {
+}
```

Comment:
return a not implemented or something for now

Reply:
this method does not return anything

Reply:
ah i mis read the signature